README.md

```diff
@@ -92,8 +102,28 @@ cdnup.init(function (err) {
 });
 ```
 
-## License
+### url
+
+Return the URL and path of the CDN.
+
+```js
+const fullCDNPath = cdnup.url();
+```
+
+### checkUrl
+
+Return the URL of the `file` specified against the configured `check`.
+
+```js
+const cdn = new CDNUp('my-bucket', {
+  check: 'https://my-bucket.s3.amazonaws.com/',
+  url: 'https://whatever.com/whatever-damnit'
```

Comment:
🤔 Probably shouldn't link to profanity in packages linked back to GoDaddy.

Reply:
lol I didn't notice, this is actually in the test file as well though